Frends.MicrosoftSQL.ExecuteQuery/README.md

```diff
@@ -1,14 +1,13 @@
 # Frends.MicrosoftSQL.ExecuteQuery
-Frends MicrosoftSQL Task to execute MSSQL query.
+Frends Task to execute Microsoft SQL Server query.
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT) 
 [![Build](https://github.com/FrendsPlatform/Frends.MicrosoftSQL/actions/workflows/ExecuteQuery_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.MicrosoftSQL/actions)
-![MyGet](https://img.shields.io/myget/frends-tasks/v/Frends.MicrosoftSQL.ExecuteQuery)
```

Comment:
Why remove this badge? Does this have something to do with the new feed?

Reply:
Jefim's comment from another review: "Remove the myget reference altogether, we are dropping it in readmes."